stdlib/public/Concurrency/TaskSleepDuration.swift

```diff
@@ -169,13 +172,18 @@ extension Task where Success == Never, Failure == Never {
   public static func sleep<C: Clock>(
     until deadline: C.Instant,
     tolerance: C.Instant.Duration? = nil,
-    clock: C
+    clock: C = ContinuousClock()
```

Comment:
I _believe_ that adding a default argument is not actually possible here, because the symbol that provides the default will not be available in library versions that predate the appearance of the default. @slavapestov, is that right?

If so, you should instead add an `@_aEIC` overload that does not take a `clock` parameter and calls this function with `clock: ContinuousClock()`

Reply:
I might be misremembering, but I'd thought that it was the longstanding case that default arguments are emitted into the client and (thus) ABI stable to add?

Reply:
That would be great if true.

Reply:
Ah, here we go: https://github.com/apple/swift/issues/56859#issuecomment-1108448735